etc/notes/CHANGES_6.0.0.md

```diff
@@ -2,23 +2,339 @@
+  - [db.command() and admin.command() unsupported options removed](#dbcommand-and-admincommand-unsupported-options-removed)
+  - [Removed irrelevant fields from `ConnectionPoolCreatedEvent.options`](#removed-irrelevant-fields-from-connectionpoolcreatedeventoptions)
+  - [Fixed parsing of empty readPreferenceTags in connection string](#fixed-parsing-of-empty-readpreferencetags-in-connection-string)
+  - [Corrected `GridFSBucketWriteStream`'s `Writable` method overrides and event emission](#corrected-gridfsbucketwritestreams-writable-method-overrides-and-event-emission)
+  - [Fix manually emitted events from `GridFSBucketReadStream`](#fix-manually-emitted-events-from-gridfsbucketreadstream)
+  - [`createDataKey` return type fix](#createdatakey-return-type-fix)
+- [📜 Removal of deprecated functionality](#-removal-of-deprecated-functionality)
+  - [`db.addUser()` and `admin.addUser()` removed](#dbadduser-and-adminadduser-removed)
+  - [`collection.stats()` removed](#collectionstats-removed)
+  - [`BulkWriteResult` deprecated properties removed](#bulkwriteresult-deprecated-properties-removed)
+  - [Deprecated SSL options have been removed](#deprecated-ssl-options-have-been-removed)
+  - [The deprecated `keepAlive` and `keepAliveInitialDelay` options have been removed](#the-deprecated-keepalive-and-keepaliveinitialdelay-options-have-been-removed)
+- [🗑️ Removal of "dead" code](#-removal-of-dead-code)
+  - [`MongoError` and its subclasses now have internal constructors](#mongoerror-and-its-subclasses-now-have-internal-constructors)
+  - [`AutoEncrypter` and `MongoClient.autoEncrypter` are now internal](#autoencrypter-and-mongoclientautoencrypter-are-now-internal)
+  - [`ClientEncryption.onKMSProvidersRefresh` function removed](#clientencryptiononkmsprovidersrefresh-function-removed)
+  - [`EvalOptions` removed](#evaloptions-removed)
+- [⚠️ ALL BREAKING CHANGES](#%EF%B8%8F-all-breaking-changes)
 
-## Changes
+## 🛠️ Runtime and dependency updates
 
-### Deprecated SSL options removed
+### Minimum Node.js version is now v16.20.1
 
-The following deprecated SSL/TLS options have now been removed (-> indicating the corresponding option):
+The minimum supported Node.js version is now v16.20.1. We strive to keep our minimum supported Node.js version in sync with the runtime's [release cadence](https://nodejs.dev/en/about/releases/) to keep up with the latest security updates and modern language features.
 
-  - `sslCA` -> `tlsCAFile`
-  - `sslCRL`
-  - `sslCert` -> `tlsCertificateKeyFile`
-  - `sslKey` -> `tlsCertificateKeyFile`
-  - `sslPass` -> `tlsCertificateKeyFilePassword`
-  - `sslValidate` -> `tlsAllowInvalidCertificates`
-  - `tlsCertificateFile` -> `tlsCertificateKeyFile`
\ No newline at end of file
+### BSON version 6.0.0
+
+This driver version has been updated to use `bson@6.0.0`. BSON functionality re-exported from the driver is subject to the changes outlined in the [BSON V6 release notes](https://github.com/mongodb/js-bson/releases/tag/v6.0.0).
+
```

Comment:
Perhaps add an FLE v6 section here, and link to: 
https://github.com/mongodb/libmongocrypt/releases/tag/node-v6.0.0

Reply:
I don't think so. The only reason bson is its own thing is because it's NOT a peer dependency, so it gets pulled in automatically. FLE is always a manual update.